src/operator/deconvolution-inl.h

```diff
@@ -30,6 +30,7 @@ namespace deconv {
 struct DeconvolutionParam : public dmlc::Parameter<DeconvolutionParam> {
   TShape kernel;
   TShape stride;
+  TShape dilate;
```

Comment:
No need to add parameter in Deconv layer. The dilate should be hardcoded to 1 in Deconv layer.